EIPS/eip-894.md

```diff
@@ -0,0 +1,36 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
+
+
+## Abstract
+Proposals that advocate for any state changes to the Ethereum blockchain should not be merged into the official Ethereum repositories.
```

Comment:
Does this include dust and state-clearing proposals like #158?

Reply:
It does. So we better make sure that gas prices are on the ball.